frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -80,13 +116,29 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
         <YellowExclamationTriangleIcon className="co-icon-space-r" /> {t('olm~Uninstall Operator?')}
       </ModalTitle>
       <ModalBody>
-        <Trans t={t} ns="olm">
-          This will remove Operator <strong>{{ name }}</strong> from{' '}
-          <strong>{{ namespace }}</strong>. Removing the Operator will not remove any of its custom
-          resource definitions or managed resources. If your Operator has deployed applications on
-          the cluster or configured off-cluster resources, these will continue to run and need to be
-          cleaned up manually.
-        </Trans>
+        <p>
+          <Trans t={t} ns="olm">
+            Operator <strong>{{ name }}</strong> will be removed from{' '}
+            <strong>{{ namespace }}</strong>, but any of its custom resource definitions or managed
+            resources will remain. If your Operator deployed applications on the cluster or
+            configured off-cluster resources, these will continue to run and require manual cleanup.
+          </Trans>
+        </p>
+        {canPatchConsoleOperatorConfig && enabledPlugins.length === 1 && (
+          <p>
+            <Trans t={t} ns="olm">
+              The <strong>{{ pluginName }}</strong> console plugin provided by this operator will be
```

Comment:
It looks like you removed the logic to get the plugin name but left the variable in the message.

Reply:
My bad; I only removed one.